src/main/java/seedu/address/logic/commands/CopyApplicantCommand.java

```diff
@@ -0,0 +1,76 @@
+package seedu.address.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+
+import java.awt.Toolkit;
+import java.awt.datatransfer.Clipboard;
+import java.awt.datatransfer.StringSelection;
+import java.util.List;
+
+import seedu.address.commons.core.index.Index;
+import seedu.address.commons.util.ToStringBuilder;
+import seedu.address.logic.Messages;
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.person.Applicant;
+
+/**
+ * Copies an applicant from the address book to the clipboard.
+ */
+public class CopyApplicantCommand extends Command {
+
+    public static final String COMMAND_WORD = "copyApplicant";
+    public static final String COMMAND_ALIAS = "cpa";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD
+            + ": Copies the details of the applicant identified by the index number used"
```

Comment:
Phrasing is a bit weird. Perhaps can change to : Copies the name and phone number of the applicant at the specified index to clipboard

Reply:
good catch! ive changed it